### DISCUSSION TITLE
Using Postgres and Mongo in the same project..

### DISCUSSION THREAD

**Discussion by victormimo** [prisma/prisma#8833] [Q&A]
Hey guys, 

I'm not sure if anyone has dealt with this before using Prisma. A project I'm working on is looking at data that is relational (Users, Teams, etc) as well as non-relational (large receipts, documents that vary). Has anyone been created a project on Prisma that uses 2 types of dbs? (is this even allowed somehow given there's only one schema.prisma?). The only other thing I can consider is creating the NoSQL db in a different project, this of course is not ideal as I would query the documents based on say userId from the users table.

Would appreciate any thoughts.

**Comment by ryands17** [comment]
@victormimo 👋 

Currently the only way to have multiple databases in the same project is using separate schemas, one for Postgres and one for MongoDB. This also means that you will have two clients to interact with.

There's a request here for the same that you can add a 👍 on and follow for more updates:
https://github.com/prisma/prisma/issues/2443

**Reply by victormimo** [reply]
Thank you @ryands17 !

I think for now I'll just keep it in one schema and work with JSON, JSONB columns.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
